app/page.tsx

```
@@ -6,7 +6,9 @@ import { AnimatedText } from "@/components/animations/animated-text";
 import { CountUp } from "@/components/animations/count-up";
 import { DashedHero } from "@/components/Hero/dashed-hero";
 import { Footer } from "@/components/Footer/footer";
-import CPUComponent from "@/components/CPU";
+import CPUComponent from "@/components/PowerBy";
```

ESLint

[error] 'CPUComponent' is defined but never used.

(@typescript-eslint/no-unused-vars)